### PR TITLE
#164198188 Users should be able to like and dislike article

### PR DESCRIPTION
**Pull request by fejiroofficial**
**What does this PR do?**
This is PR for building a feature that allows users to like and dislike articles.

**Description of Task to be completed?**
- Write unit test(s)
- Write code and migration scripts
- Write Controller functions and validators to handle feature

**How should this be manually tested**
- Clone this repo and cd into it
- run `npm install` to install dependencies.
- test using mocha with `npm test`.

**What are the relevant pivotal tracker stories**
[#164198188](https://www.pivotaltracker.com/n/projects/2313263/stories/164198188)

**Screenshots**
N/A

### CODE REVIEW

File: middlewares/articlesMiddleware.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default class AriclesMiddleware {
       const foundArticle = await articleHelpers.findArticleByAuthorId(authorId, title);
       if (foundArticle && foundArticle.title === title) {
         return Response.send(
-          res, 400, [], 'an article with that title already exist', 'failed',
+          res, 403, [], 'an article with that title already exist', 'failed',
```
Comment (aanchirinah) [inline middlewares/articlesMiddleware.js]: #### Nitpick
- Thought there was a constants file for status codes

File: test/integrations/routes/articles.test.js [hunk 2]
```diff
@@ -13,18 +12,21 @@ chai.use(chaiHttp);
...
+
 const dummyUser = {
   email: 'faker@email.com',
   password: 'i2345678',
   username: 'heron419'
 };
 
 const dummyArticle = {
-  title: 'Hello world',
+  title: 'Hello worlrd',
```
Comment (aanchirinah) [inline test/integrations/routes/articles.test.js]: #### Nitpick
- Spelling check

File: test/integrations/routes/articles.test.js [hunk 3]
```diff
@@ -44,37 +47,50 @@ before(async () => {
...
+      });
+      it('Should return an error', (done) => {
         chai
           .request(app)
           .post('/api/v1/articles')
           .send(dummyArticle)
           .set({ Authorization: `Bearer ${authToken}` })
           .end((err, res) => {
-            expect(res).to.have.status(STATUS.BAD_REQUEST);
+            expect(res).to.have.status(403);
```
Comment (aanchirinah) [inline test/integrations/routes/articles.test.js]: #### Nitpick
- Is the team doing away with using constants?

File: middlewares/articlesMiddleware.js (hunk 1 above)
Comment (thislekan) [reply middlewares/articlesMiddleware.js]: Hello Audax. The constants file requires the status code as parameters. What he's done is to pass it the required parameters.

**Comment by thislekan** [comment]
LGTM

**Pull request merged**